SensorTemperaturaHumedad/DHT_MQTT/src/main.cpp

```diff
@@ -31,7 +38,23 @@ const char *publish = "";
 const char* wifi_ssid = "";
 const char* password = "";
 
-//setting a WifiClient
+
+char topic[150];
+char payload[50];
+String clientMac = "";
+unsigned char mac[6];
+
+struct Config {
+   /* data */
+  char name[20];
+  bool enabled;
+  int hour;
+} config;
```

Comment:
line 52, what does "**config;**" do? it looks weird to be next to **} ** symbol

Reply:
is am enum [see more here](https://www.programiz.com/cpp-programming/enumeration).  Is used in C++